README.md

```diff
@@ -14,51 +14,31 @@ You can find Aiia documentation at <https://docs.aiia.eu/>
 
 ### Running locally
 
-For starters, you'll need to create an account at <https://portal.aiia.eu/> and generate a `ClientId` and `ClientSecret`.
+1) You'll need to create an account at <https://portal.aiia.eu/> and generate a `ClientId` and `ClientSecret`.
+  1) As a webhook url specify https://<hostname>/aiia/webhook if your website is publicly reachable. If this is not the case it can also be left empty, but you won't receive any webhook in that case.
+  1) As a redirect url https://<hostname>/aiia/callback. You can also use localhost for this field (remember the port number!)
```

Comment:
did you want to have these 2 points indented? github's markdown preview shows a flat hierarchy for all points in this list

Reply:
Yes I wanted that - let me try fixing it :(